rest/taskana-rest-spring-test/src/test/java/pro/taskana/rest/ClassificationDefinitionControllerIntTest.java

```diff
@@ -358,7 +359,7 @@ public void testChangeParentByImportingExistingClassification() throws IOExcepti
         ResponseEntity<String> response = importRequest(clList);
         assertEquals(HttpStatus.OK, response.getStatusCode());
         Thread.sleep(10);
-        LOGGER.debug("Wait for 10 s to give the system a chance to update");
```

Comment:
Warum hast du das geändert? In diesem Fall finde ich es hilfreich zu wissen, das hier eine geplant Pause von 10 Sekunden passiert.